backend/src/main/java/com/google/capstone/TextToSpeech.java

```diff
@@ -67,14 +70,20 @@ private ByteString generateAudio(TextToSpeechClient textToSpeechClient, String t
             AudioConfig.newBuilder().setAudioEncoding(AudioEncoding.MP3).build();
     SynthesizeSpeechResponse ttsResponse =
             textToSpeechClient.synthesizeSpeech(input, voice, audioConfig);
+    logger.info("Text successfully synthesized to speech.");
     return ttsResponse.getAudioContent();
   }
 
   private void uploadAudio(ByteString audioContents, HttpServletResponse response, String objectIdString) throws IOException {
     Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
     BlobId blobId = BlobId.of(BUCKET_NAME, objectIdString);
     BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("audio/mpeg").build();
-    storage.create(blobInfo, audioContents.toByteArray());
+    try {
+      storage.create(blobInfo, audioContents.toByteArray());
+    } catch (StorageException e) {
+      logger.severe("Audio file unable to be uploaded to Google Cloud Storage.");
+    }
+    logger.info("Audio file successfully uploaded to Google Cloud Storage.");
```

Comment:
Where does this logger.info go? From line 34 I see it's related to Text-to-Speech so is it read out?

Reply:
This is just information that will be in the logs. Line 34 initializes the logger for this class.